dockerfiles/alpine/is/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # ------------------------------------------------------------------------
 #
-# Copyright 2021 WSO2, Inc. (http://wso2.com)
+# Copyright 2024 WSO2, LLC. (http://wso2.com)
```

Comment:
Shall we add the year range instead of changing to 2024?

Reply:
Addressed with https://github.com/wso2/docker-is/pull/410/commits/5c793fd240777975094dbfe814c373624493a0c2